src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs

```diff
@@ -1221,6 +1222,16 @@ public Message Request(Message message, TimeSpan timeout)
             return (Message)Call(message.Headers.Action, false, operation, new object[] { message }, Array.Empty<object>(), timeout);
         }
 
+        public virtual Task<Message> RequestAsync(Message message)
```

Comment:
There's no requirement for these to be virtual. The reason many classes have Async methods as virtual is to prevent breaking existing code which derives from the class which might not have overridden the Async methods. As this class was sealed, nothing derives from it (and it's internal) so there's no need to make the methods virtual.